adapter/service/parameters/event_test.go

```diff
@@ -0,0 +1,164 @@
+package parameters_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/futurehomeno/fimpgo"
+	"github.com/futurehomeno/fimpgo/fimptype"
+
+	"github.com/futurehomeno/cliffhanger/adapter"
+	"github.com/futurehomeno/cliffhanger/adapter/service/chargepoint"
+	"github.com/futurehomeno/cliffhanger/adapter/service/parameters"
+	"github.com/futurehomeno/cliffhanger/discovery"
+	"github.com/futurehomeno/cliffhanger/event"
+	"github.com/futurehomeno/cliffhanger/lifecycle"
+	"github.com/futurehomeno/cliffhanger/root"
+	"github.com/futurehomeno/cliffhanger/router"
+	adapterhelper "github.com/futurehomeno/cliffhanger/test/helper/adapter"
+	mockedadapter "github.com/futurehomeno/cliffhanger/test/mocks/adapter"
+	mockedchargepoint "github.com/futurehomeno/cliffhanger/test/mocks/adapter/service/chargepoint"
+	mockedparameters "github.com/futurehomeno/cliffhanger/test/mocks/adapter/service/parameters"
+	"github.com/futurehomeno/cliffhanger/test/suite"
+)
+
+const (
+	testServiceName = "test_adapter"
+)
+
+func TestParametersReport(t *testing.T) { //nolint:paralleltest
+	s := &suite.Suite{
+		Cases: []*suite.Case{
+			{
+				Name:     "Supported parameters report after inclusion report",
+				TearDown: adapterhelper.TearDownAdapter("../../testdata/adapter/test_adapter"),
+				Setup:    testSetup(true),
+				Nodes: []*suite.Node{
+					{
+						Name:    "Thing with parameters service",
+						Command: suite.StringMessage("pt:j1/mt:cmd/rt:ad/rn:test_adapter/ad:1", "cmd.thing.get_inclusion_report", testServiceName, "1"),
```

Comment:
If you want to stick to the const for test_adapter, please be consequent and use it everywhere.